interface/patient_file/summary/immunizations.php

```diff
@@ -93,6 +95,10 @@
         if ($GLOBALS['observation_results_immunization']) {
             saveImmunizationObservationResults($newid, $_GET);
         }
+        if ($GLOBALS['inhouse_pharmacy'] > 0) {
+            $decreaseInventoriedItem = new InventoryDecrementEvent($_GET['lot_number']);
```

Comment:
hi @juggernautsei , have you tried the easy development docker to easily spin up a branch for testing? It should trigger this bit of code and throw a syntax error for missing a `)` at the end of the expression.

`             $decreaseInventoriedItem = new InventoryDecrementEvent($_GET['lot_number']));`

Reply:
wait, it's not missing anything there, sorry

Reply:
Thanks, @stephenwaite. I will fix that syntax error.